### PR TITLE
2.13 release notes

### PR DESCRIPTION
**Pull request by scrawfor99**
### Description
[Describe what this change achieves]
Add release notes for 2.13 release (https://github.com/opensearch-project/security-dashboards-plugin/issues/1774)

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,5 +45,6 @@
     "selenium-webdriver": "4.10.0",
     "glob-parent": "^5.1.2",
     "debug": "^4.3.4"
-  }
-}
\ No newline at end of file
+  },
```
Comment (peternied) [inline package.json:48]: Lets revert this files change

File: release-notes/opensearch-security-dashboards-plugin.release-notes-2.13.0.0.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## 2024-03-19 Version 2.13.0.0
+
+Compatible with OpenSearch-Dashboards 2.13.0
+
+### Enhancements
+* Clear the contents of opensearch_dashboards prior to putting settings ([#1781](https://github.com/opensearch-project/security-dashboards-plugin/pull/1781))
+* Add loose flag to OSD bootstrap ([#1789](https://github.com/opensearch-project/security-dashboards-plugin/pull/1789))
+* Hide tenant when disabled in the account nav button popover ([#1792](https://github.com/opensearch-project/security-dashboards-plugin/pull/1792))
+*  Use start-opensearch and setup-opensearch-dashboards actions ([#1808](https://github.com/opensearch-project/security-dashboards-plugin/pull/1808))
```
Comment (peternied) [inline release-notes/opensearch-security-dashboards-plugin.release-notes-2.13.0.0.md:9]: Extra space, can you removed?

File: package.json (hunk 1 above)
Comment (scrawfor99) [reply package.json:48]: Hmmm, oops this what unintentional

File: release-notes/opensearch-security-dashboards-plugin.release-notes-2.13.0.0.md (hunk 2 above)
Comment (scrawfor99) [reply release-notes/opensearch-security-dashboards-plugin.release-notes-2.13.0.0.md:9]: Thanks good catch.

**Review by derek-ho — APPROVED** [review]
LGTM

**Review by DarshitChanpura — APPROVED** [review]

File: release-notes/opensearch-security-dashboards-plugin.release-notes-2.13.0.0.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+## 2024-03-19 Version 2.13.0.0
+
+Compatible with OpenSearch-Dashboards 2.13.0
+
+### Enhancements
+* Clear the contents of opensearch_dashboards prior to putting settings ([#1781](https://github.com/opensearch-project/security-dashboards-plugin/pull/1781))
+* Add loose flag to OSD bootstrap ([#1789](https://github.com/opensearch-project/security-dashboards-plugin/pull/1789))
+* Hide tenant when disabled in the account nav button popover ([#1792](https://github.com/opensearch-project/security-dashboards-plugin/pull/1792))
+* Use start-opensearch and setup-opensearch-dashboards actions ([#1808](https://github.com/opensearch-project/security-dashboards-plugin/pull/1808))
+* Fix cookie expiry issues from IDP/JWT auth methods, disables keepalive for JWT/IDP ([#1806](https://github.com/opensearch-project/security-dashboards-plugin/pull/1806))
```
Comment (cwperks) [inline release-notes/opensearch-security-dashboards-plugin.release-notes-2.13.0.0.md:10]: nit: I would organize some of these as Bug Fixes or maintenance.

**Review by cwperks — APPROVED** [review]

**Pull request merged**